frontend/app/components/import_report/import_report.component.html

```diff
@@ -1,334 +1,292 @@
-                </mat-panel-title>
-              </mat-expansion-panel-header>
-              <table class="table table-striped table-bordered">
-                <thead>
-                  <tr>
-                    <th>Champ source</th>
-                    <th>Champ cible</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  <tr *ngFor="let field of importData?.fieldmapping | keyvalue ">
-                    <td>{{ field.value }}</td>
-                    <td>{{ field.key }}</td>
-                  </tr>
-                </tbody>
-              </table>
-            </mat-expansion-panel>
-            <mat-expansion-panel>
-              <mat-expansion-panel-header [collapsedHeight]="expansionPanelHeight">
-                <mat-panel-title>
-                  <h6>
-                    Nomenclatures ({{ (importData?.contentmapping || {} | keyvalue).length }}
-                    type(s de nomenclatures))
-                  </h6>
-                </mat-panel-title>
-              </mat-expansion-panel-header>
-              <table class="table table-striped table-bordered">
-                <thead>
-                  <tr>
-                    <th>Valeur source</th>
-                    <th>Nomenclature</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  <ng-container *ngFor="let nomenclature_type of importData?.contentmapping | keyvalue">
-                  <tr>
-                    <th colspan="2">
-                      <ng-container *ngIf="nomenclatures && nomenclatures.hasOwnProperty(nomenclature_type.key); then nomenclature_type_label else nomenclature_type_mnemonique"></ng-container>
-                      <ng-template #nomenclature_type_label>{{ nomenclatures[nomenclature_type.key].nomenclature_type.label_default }}</ng-template>
-                      <ng-template #nomenclature_type_mnemonique>{{ nomenclature_type.key }}</ng-template>
-                    </th>
-                  </tr>
-                  <tr *ngFor="let mapping of nomenclature_type.value | keyvalue">
-                    <td>{{ mapping.key }}</td>
-                    <td>
-                      <ng-container *ngIf="nomenclatures && nomenclatures.hasOwnProperty(nomenclature_type.key); then nomenclature_label else nomenclature_code"></ng-container>
-                      <ng-template #nomenclature_label>{{ nomenclatures[nomenclature_type.key].nomenclatures[mapping.value].label_default }}</ng-template>
-                      <ng-template #nomenclature_code>{{ mapping.value }}</ng-template>
-                    </td>
-                  </tr>
-                  </ng-container>
-                </tbody>
-              </table>
-            </mat-expansion-panel>
+    </div>
+  <div class="card-body">
+    <mat-expansion-panel class="card content" [expanded]="true">
+      <mat-expansion-panel-header class="card-header">
+        <mat-panel-title>
+        <mat-icon>build</mat-icon>
```

Comment:
The ``build`` icon should be replaced by the ``info`` icon. Thanks.

Reply:
OK to replace the icon

Reply:
Done